loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowParametersDeserializer.java

```diff
@@ -103,6 +103,18 @@ public LoadFlowParameters deserialize(JsonParser parser, DeserializationContext
                     parameters.setWriteSlackBus(parser.readValueAs(Boolean.class));
                     break;
 
+                case "voltageRemoteControl":
+                    JsonUtil.assertGreaterThanReferenceVersion(CONTEXT_NAME, "Tag: voltageRemoteControl", version, "1.3");
```

Comment:
1.4?

Reply:
1.3 is right.
Check version is strickly superior to 1.3